ui/src/app/pages/datasets/dataset-from/dataset-from.component.ts

```diff
@@ -91,52 +106,59 @@ export class DatasetFromComponent implements OnInit {
     } as Dataset;
 
     this.form = this._formBuilder.group({
-      name: [name, [Validators.required, Validators.pattern('^[a-zA-Z_][a-zA-Z0-9_-]*$')]],
-      agent_policy_id: [agent_policy_id, [Validators.required]],
-      agent_group_id: [agent_group_id, [Validators.required]],
-      sink_ids: [sink_ids, [Validators.minLength(1)]],
-    });
+                                          name: [name, [Validators.required, Validators.pattern(
+                                            '^[a-zA-Z_][a-zA-Z0-9_-]*$')]],
+                                          agent_policy_id: [agent_policy_id, [Validators.required]],
+                                          agent_group_id: [agent_group_id, [Validators.required]],
+                                          sink_ids: [sink_ids, [Validators.minLength(1)]],
+                                        });
   }
 
   ngOnInit(): void {
     if (!!this.group) {
       this.selectedGroup = this.group.id;
-      this.form.patchValue({agent_group_id: this.group.id});
+      this.form.patchValue({ agent_group_id: this.group.id });
     }
     if (!!this.policy) {
       this.selectedPolicy = this.policy.id;
-      this.form.patchValue({agent_policy_id: this.policy.id});
+      this.form.patchValue({ agent_policy_id: this.policy.id });
     }
     if (!!this.dataset) {
-      const {name, agent_group_id, agent_policy_id, sink_ids} = this.dataset;
+      const { name, agent_group_id, agent_policy_id, sink_ids } = this.dataset;
       this.selectedGroup = agent_group_id;
-      this.selectedSinks = this.availableSinks.filter(sink => sink_ids.indexOf(sink.id) !== -1);
+      this._selectedSinks = this.availableSinks.filter(sink => sink_ids.includes(sink.id));
       this.selectedPolicy = agent_policy_id;
-      this.form.patchValue({name, agent_group_id, agent_policy_id, sink_ids});
+      this.form.patchValue({ name, agent_group_id, agent_policy_id, sink_ids });
       this.isEdit = true;
+
+      this.unselectedSinks = this.availableSinks.filter(sink => !this._selectedSinks.includes(sink));
     }
   }
 
+  updateUnselectedSinks() {
+    this.unselectedSinks = this.availableSinks.filter(sink => !this._selectedSinks.includes(sink));
```

Comment:
tldr; Yes.

---
I need to create a PR applying the correct format to all source.

Since updating Jetbrains account, I've lost the previous project file which contained the formatting.

I have since done some improvements to it I'd like to apply to all codebase too.

:-1:

Reply:
pushed a format with hard line break @80